lib/node_modules/@stdlib/_tools/scaffold/math-special-unary/scripts/runner.js

```diff
@@ -0,0 +1,479 @@
+*
+* @private
+* @param {Object} obj - configuration object
+* @returns {string} environment variable string
+*
+* @example
+* var o = {
+*     'prng': 'random/base/uniform',
+*     'parameters': [ -10, 10 ]
+* };
+*
+* var s = prngEnvVars( o );
+* // returns 'BASE_PRNG=\'uniform\',RAND_MIN=-10,RAND_MAX=10'
+*/
+function prngEnvVars( obj ) {
+	var base;
+	var min;
+	var max;
+	var p;
+
+	p = obj.prng.split( '/' );
+	base = p[ p.length-1 ];
+	min = obj.parameters[ 0 ];
+	max = obj.parameters[ 1 ];
+	if ( base === 'uniform' ) {
+		if ( isInteger( min ) ) {
+			min = min.toString() + '.0';
+		}
+		if ( isInteger( max ) ) {
+			max = max.toString() + '.0';
+		}
+	} else {
+		min = min.toString();
+		max = max.toString();
+	}
+	return [
+		'BASE_PRNG=\'' + base + '\'',
+		'RAND_MIN=' + min,
+		'RAND_MAX=' + max
+	].join( ' ' );
+}
+
+/**
+* Returns an `ALIAS` environment variable string.
+*
+* @private
+* @param {string} alias - package alias
+* @returns {string} environment variable string
+*
+* @example
+* var s = aliasEnvVar( 'abs' );
+* // returns 'ALIAS=abs'
+*
+* @example
+* var s = aliasEnvVar( 'absf' );
+* // returns 'ALIAS=absf'
+*/
+function aliasEnvVar( alias ) {
+	return 'ALIAS=' + alias;
+}
+
+/**
+* Returns a string containing example values for a 2x2 array environment variable.
+*
+* @private
+* @param {Array} values - example values
+* @returns {string} environment variable string
+*
+* @example
+* var exampleValues = [ 1.9, 4.8, 6.0, 8.1 ];
+* var out = jsExampleValues2x2EnvVar( exampleValues );
+* // returns "JS_EXAMPLE_VALUES_2X2='[ [ 1.9, 4.8 ], [ 6.0, 8.1 ] ]'"
+*/
+function jsExampleValues2x2EnvVar( values ) {
+	var bottomRow;
+	var topRow;
+	var vals;
+	var v;
+	var i;
+
+	vals = [];
+	for ( i = 0; i < 4; i++ ) {
+		v = values[ i ];
+		vals.push( v.toString() );
+	}
+	topRow = join( slice( vals, 0, 2 ), ', ' );
+	bottomRow = join( slice( vals, 2, 4 ), ', ' );
+	return 'JS_EXAMPLE_VALUES_2X2=\'[ [ ' + topRow + ' ], [ ' + bottomRow + ' ] ]\'';
+}
+
+/**
+* Returns a string containing example results for a 2x2 array environment variable.
+*
+* @private
+* @param {Array} values - input values
+* @param {string} functionAlias - function alias (e.g., 'sin', 'sqrt', 'abs')
+* @returns {string} environment variable string
+*
+* @example
+* var exampleValues = [ 0.0, 1.57, 3.14, 4.71 ];
+* var out = jsExampleResults2x2EnvVar( exampleValues, 'sin' );
+* // returns "JS_EXAMPLE_RESULTS_2X2='[ [ 0.0, 1.0 ], [ -0.0, -1.0 ] ]'"
+*/
+function jsExampleResults2x2EnvVar( values, functionAlias ) {
+	var bottomRow;
+	var results;
+	var topRow;
+	var func;
+	var vals;
+	var v;
+	var i;
+
+	func = require( '@stdlib/math/base/special/' + functionAlias ); // eslint-disable-line stdlib/no-dynamic-require
+	vals = [];
+	for ( i = 0; i < 4; i++ ) {
+		v = values[ i ];
+		results = func( v );
+		results = round( results * 100 ) / 100; // Rounding results to 2 decimal places
+		vals.push( '~' + results.toString() );
```

Comment:
I think you only want to round if the result is not exact (i.e., an integer value). Otherwise, if results are exact, you don't need a `~`. Furthermore, if integer-valued, you also need to include trailing `.0` characters to the results if the array type is floating-point.